README.md

```diff
@@ -17,6 +17,23 @@ for Ada:
 
 These are currently under development.
 
+Installing
+----------
+If you are only concerned in getting the suite of executables you can most easily do so by using the [Alire][alire] package manager to handle the building and installing for you. The steps are:
+```bash
+alr get libadalang_tools
+cd libadalang_tools
+alr install
```

Comment:
This can probably be replaced with a single line? Ref https://alire.ada.dev/docs/#new-subcommand-alr-install
```suggestion
alr install libadalang_tools
```